projects/vdk-plugins/vdk-jupyter/convert-job-tutorial-temp/tutorial.md

```diff
@@ -0,0 +1,86 @@
+Learn how to convert a VDK Job with .sql and .py steps into a Job with notebook steps using VDK Jupyter Notebook UI.
+Time Commitment: About 5-10 minutes.
+
+## Prerequisites
+* Installed VDK, Control Service, Jupyter instance, vdk-jupyterlab-extension, vdk-notebook, vdk-ipython
+* Created and developed a VDK Data Job
+
+## Data Job Conversion Guide
+
+This guide presumes that you have already initialized a job either locally or in the cloud and have subsequently
+downloaded it to your local file system.
+
+### 0. [Optional] Get Familiar With Your Data Job
+- **Navigate to the data job directory.**
+- Familiarize yourself with the VDK steps and the file structure of the Data Job.
+
+  ![VDK Initial Job Directory](images/initial_job_dir.png)
+
+### 1. [Optional] Validate Your Data Job's Functionality
```

Comment:
This really seems more like a Tip like https://github.com/vmware/versatile-data-kit/wiki/Tutorial-Guidelines#2-content